ImageChecks.ps1

```diff
@@ -0,0 +1,167 @@
+param (
+    [String]$Image,
+    [String]$ConfigFile
+)
+
+[String]$resourcesDir = "$pwd\UnattendResources"
+[String]$imgExtracted
+$extension=[IO.Path]::GetExtension($Image)
+$extension=$extension.split(".")[1]
+
+function Get-Type {

```

Comment:
This is a function name that can be confusing, for what are you trying to get the type of?

Get-ImageType sounds better